src/rules/stylistic-issues.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
         'camelcase': ['error', {                                                                  // enforce camelcase naming convention
             properties: 'never',
             ignoreDestructuring: true,
-            allow: ['^UNSAFE_'],
+            allow: ['^UNSAFE_', '^[a-zA-Z0-9-]+_{2}[a-zA-Z0-9-]+$'],
```

Comment:
this regex is to allow JSX naming like SomeComponent__SubComponent

Reply:
put that comment inline next to the rule